frontend/tests/features/collection.test.ts

```diff
@@ -0,0 +1,57 @@
+import { ROUTES } from "src/common/constants/routes";
+import { apiTemplateToUrl } from "src/common/utils/apiTemplateToUrl";
+import { API_URL } from "src/configs/configs";
+import { TEST_ENV, TEST_URL } from "tests/common/constants";
+import { goToPage } from "tests/utils/helpers";
+import { getText } from "tests/utils/selectors";
+
+const describeIfCalledByDevEnv = TEST_ENV === "dev" ? describe : describe.skip;
+
+/*  eslint-disable sort-keys */
+const TEST_COLLECTION = {
+  name: "TEST COLLECTION",
+  description: "TEST DESCRIPTION",
+  contactName: "TEST NAME",
+  contactEmail: "TEST@example.com",
+};
+/* eslint-enable sort-keys */
+
+describeIfCalledByDevEnv("Collection", async () => {
+  it("creates and deletes a collection", async () => {
+    await goToPage(TEST_URL);
+    // LOG IN
+    await page.click(getText("Log In"));
```

Comment:
do we need to type email and password to log in?

Reply:
This is assuming we're using Trent's mock auth which will just give us an auth token with no user input needed

Reply:
Ahh I see! That's super awesome!